### DISCUSSION TITLE
Original Stand Up Coaster/Trains from RCT 1 (Not the twister coaster varient)

### DISCUSSION THREAD

**Discussion by Mogoroth** [OpenRCT2/OpenRCT2#20613] [Feature requests] [answered]
To add in the original stand up coaster (or the trains from it) from RCT1. 

![RCT1STANDUPCOASTER](https://github.com/OpenRCT2/OpenRCT2/assets/139267122/40ca07ba-0276-45e9-abb2-0aba60e6e2b1)

**Comment by SpartanFrederic104** [comment, marked as answer]
There's already a discussion for this: #18171
